ceph-dev-setup/build/build

```diff
@@ -43,25 +43,44 @@ echo "Running submodule update ..."
 git submodule update --init
 
 # Flavor Builds support
-# TODO: enable CMAKE
-#export CEPH_EXTRA_CMAKE_ARGS
 
-FLAVOR_FLAGS="--without-cryptopp"
```

Comment:
$FLAVOR_FLAGS is still being used later in the script at https://github.com/ceph/ceph-build/blob/wip-cmake/ceph-dev-setup/build/build#L146